encoder_test.go

```diff
@@ -523,3 +523,20 @@ func TestRegisterEncoderWithPtrType(t *testing.T) {
 	valExists(t, "DateStart", ss.DateStart.time.String(), vals)
 	valExists(t, "DateEnd", "", vals)
 }
+
+func TestNewEncoderWithOptions(t *testing.T) {
+	defaultEncoder := NewEncoder()
+
+	aliasTag := defaultEncoder.cache.tag + "-test"
```

Comment:
NIT: `defaultEncoder.cache.tag` will be always equal to "schema" because it's hard coded: https://github.com/gorilla/schema/blob/main/cache.go#L22. By creating a constant we can avoid instantiating `defaultEncoder := NewEncoder()`